### PR TITLE
Fix minimize

### PR DESCRIPTION
**Pull request by IslandRhythms**

### CODE REVIEW

**Comment by IslandRhythms** [comment]
The failing linter test is weird because that line that it is referencing has no code on it.

File: lib/document.js [hunk 1]
```diff
@@ -3406,20 +3396,22 @@ Document.prototype.$toObject = function(options, json) {
...
-
   let _minimize;
+
+  // console.log('Schema Options', schemaOptions.minimize)
+  // console.log('options', options)
   if (options._calledWithOptions.minimize != null) {
     _minimize = options.minimize;
   } else if (defaultOptions.minimize != null) {
     _minimize = defaultOptions.minimize;
+  } else if (options._parentOptions != null && options.getters != null) {
```
Comment (vkarpov15) [inline lib/document.js:3410]: Why `options.getters != null` here?

File: lib/document.js [hunk 2]
```diff
@@ -3406,20 +3396,22 @@ Document.prototype.$toObject = function(options, json) {
...
   defaultOptions = utils.options(defaultOptions, clone(baseOptions));
   defaultOptions = utils.options(defaultOptions, clone(schemaOptions[path] || {}));
-
   // If options do not exist or is not an object, set it to empty object
   options = utils.isPOJO(options) ? clone(options) : {};
   options._calledWithOptions = options._calledWithOptions || clone(options);
-
   let _minimize;
+
+  // console.log('Schema Options', schemaOptions.minimize)
```
Comment (vkarpov15) [inline lib/document.js:3404]: Get rid of the unnecessary commented out code please

**Review by vkarpov15 — COMMENTED** [review]
Don't worry about the lint failure, I'll fix that in master

File: lib/document.js (hunk 1 above)
Comment (IslandRhythms) [reply lib/document.js:3410]: it won't work without it. gh-9405 has tests already and this was the only combination that could get them all to pass.

**Comment by vkarpov15** [comment]
I took a closer look and this is expected behavior. We made the change in #9405 that subdocs inherit parents' minimize and added tests that enforce that behavior, so no way to make this work. The `getters` trick works by accident rather than by design.

If you find yourself struggling to reconcile behavior that's asserted on in the tests with new behavior, feel free to ask me to clarify in Slack.

**Pull request closed without merging**